src/Polly.Strategies/Timeout/TimeoutResilienceStrategy.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.Extensions.ObjectPool;
+using Polly.Internals;
+
+namespace Polly.Timeout;
+
+internal sealed class TimeoutResilienceStrategy : DelegatingResilienceStrategy
+{
+    private static readonly ObjectPool<CancellationTokenSource> _cancellations = ObjectPool.Create<CancellationTokenSource>();
```

Comment:
In another class you prefixed this `s_`. Personally I'd prefer without the `s`, but whichever choice it would be better to be consistent.

Reply:
I prefer `_` to, so renamed the other one.